css/styles.css

```diff
@@ -320,3 +424,79 @@ label {
 #homeDesktop:hover {
   filter: drop-shadow(0 2px 4px rgba(0, 0, 0, 0.25));
 }
+
+#addVB {
+  position: fixed;
+  right: 8%;
+  bottom: 5%;
+  background-color: #007bff;
+  width: 70px;
+  height: 70px;
+  border-radius: 50%;
+}
+
+#iAddVB {
+  -webkit-text-stroke: none;
+  -webkit-text-fill-color: white;
+}
+
+#cancel {
+  background-color: #909090;
+  top: 22px;
+  left: 0;
+}
+
+#delete {
+  background-color: #d22323;
+  top: 22px;
+  left: 35%;
+}
+
+#confirm {
+  background-color: #007bff;
+  top: 22px;
+  right: 0;
+}
```

Comment:
Why is positioning needed here? Don't these buttons just sit next to each other?